### PR TITLE
Updating PPO link in README

### PR DESCRIPTION
**Pull request by SalmanMohammadi**
Thanks to @ebsmothers for updating the README and the shout-outs in #1267.

I'm so sorry, I was distracted by the comments and forgot to properly review the PR. The link to the PPO recipe pointed to DPO. 
I think a lot of people also use RLHF to refer to the technique so updated this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -282,7 +282,7 @@ torchtune embodies PyTorch’s design philosophy [[details](https://pytorch.org/
 
 ### Native PyTorch
 
-torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (eg: Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
+torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (e.g: Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
```
Comment (joecummings) [inline README.md:285]:
```suggestion
torchtune is a native-PyTorch library. While we provide integrations with the surrounding ecosystem (e.g. Hugging Face Datasets, EleutherAI Eval Harness), all of the core functionality is written in PyTorch.
```

Comment (SalmanMohammadi) [reply README.md:285]: 🤯🤯 💯💯 🔥🔥

**Review by RdoubleA — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (joecummings) [reply README.md]: Screenshot 2024-08-05 at 5 56 44 PM

TIL

Comment (SalmanMohammadi) [reply README.md]: image

Comment (SalmanMohammadi) [reply README.md]: slightly more esoteric;

image

Comment (SalmanMohammadi) [reply README.md]: and who knew RIP was originally latin???
image

Comment (joecummings) [reply README.md]: Not reviewing anymore RFCs unless they end in QED so I know it was *demonstrated*

**Pull request merged**